src/ngraph/op/fused/group_conv.hpp

```diff
@@ -85,5 +85,83 @@ namespace ngraph
         private:
             bool has_groups_in_filters_shape() const;
         };
+
+        /// \brief Group Convolution data batch backprop
+        class GroupConvolutionBackpropData : public ngraph::op::util::FusedOp
+        {
+        public:
+            NGRAPH_API
```

Comment:
As of this morning, it's `class NGRAPH_API GroupConvolutionBackpropData ...` instead of the `NGRAPH_API` before the `type_info`.

Reply:
updated